charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -1,8 +1,22 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: pp-plugins-scheduler
+  name: aggregated-pp-plugins-scheduler-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
```

Comment:
Could you clarify why this is being added? Looking at the [doc](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) `rbac.authorization.k8s.io/aggregate-to-view` is for user facing cluster role. Why is it required here for the scheduler plugin?

Reply:
let me try to remove it after updating the kubeconfig part

Reply:
Yup, no need for it now, removed